Source/Core/VideoCommon/TextureCacheBase.cpp

```diff
@@ -238,6 +294,8 @@ static u32 CalculateLevelSize(u32 level_0_size, u32 level)
 // Used by TextureCache::Load
 TextureCache::TCacheEntryBase* TextureCache::ReturnEntry(unsigned int stage, TCacheEntryBase* entry)
 {
+	entry->do_partial_texture_updates();
```

Comment:
This depends on the order in which things are happening, which i don't know. The big texture in new super mario bros gets changed for different worlds(or maybe even single stages). When does this happen, and when do the texture updates via efb copies happen?

Or more precise, what's there 1st? The efb copies or the texture?